tests/layer_tests/pytorch_tests/test_all.py

```diff
@@ -34,32 +34,35 @@ class TestAll(PytorchLayerTest):
     def _prepare_input(self):
         return (self.input_tensor,)
 
-    @pytest.mark.parametrize("input_tensor", [
-        np.eye(5,5),
-        np.zeros((5, 5)),
-        np.zeros((9,8)) + 1,
-        np.random.randint(0, 2, (5, 9, 7)),
-        np.random.randint(0, 2, (10, 13, 11)),
-        np.random.randint(0, 2, (8, 7, 6, 5, 4)),
-        np.random.randint(0, 2, (11, 11), dtype=np.uint8),
-        np.random.randint(0, 2, (7, 7), dtype=np.uint8),
+    @pytest.mark.parametrize("input_shape, d_type", [
+        (np.eye(5,5), np.int16),
+        (np.zeros((5, 5)), np.int16),
+        (np.zeros((9,8)) + 1, np.int16),
+        ([5, 9, 7], np.int16),
```

Comment:
Why `int16`? Previously it as `int64` which is also not great, better would be `int32`

Reply:
yes, missed that, should be int64 (as it was by default)

Reply:
fixed